themes/root/theme.config.php

```diff
@@ -121,6 +122,7 @@
             'transEsc' => 'VuFind\View\Helper\Root\TransEsc',
             'translate' => 'VuFind\View\Helper\Root\Translate',
             'truncate' => 'VuFind\View\Helper\Root\Truncate',
+            'url' => 'VuFind\View\Helper\Root\Url',
```

Comment:
It might be preferable to use:

`'Zend\View\Helper\Url' => 'VuFind\View\Helper\Root\Url',`

here to make it more explicit that we're overriding a core helper (and to ensure that any other pre-existing aliases work more consistently). For our purposes, the approaches should be functionally identical.